impl/src/metadata.rs

```diff
@@ -19,16 +19,70 @@ use std::{
   string::String,
 };
 
-use anyhow::{anyhow, Result};
+use anyhow::{anyhow, Context, Result};
 
+use cargo_lock::Lockfile;
 use cargo_metadata::MetadataCommand;
-pub use cargo_metadata::{DependencyKind, Metadata, Node, Package, PackageId};
 
+use glob::glob;
+use pathdiff::diff_paths;
+use regex::Regex;
+use rustc_serialize::hex::ToHex;
 use tempfile::TempDir;
+use url::Url;
 
-use rustc_serialize::hex::ToHex;
+// These attriburtes are the source of truth for the Metadata used by Cargo raze.
+// Making them public allows for dependents to always ensure they're using the 
+// same versions `cargo-raze` is.
+pub use cargo_metadata::{Metadata, Package, PackageId};
```

Comment:
Not sure why the comment disappeared, but RE your last response.

Lets not reexport this. If an end users needs this, we can decide to reexport it at that time once a specific case has been made.

Reply:
This has been updated and is no longer public.